docs/reference/mapping/runtime.asciidoc

```diff
@@ -164,6 +164,32 @@ PUT my-index/
 }
 ----
 
+When no script is provided, {es} implicitly looks in `_source` at query time
+for a field with the same name as the runtime field, and returns a value if one
+exists. If a field with the same name doesn’t exist, the response doesn't
+include any values for that runtime field.
+
+In most cases, retrieve field values through
+<<doc-values,`doc_values`>> whenever possible. Accessing `doc_values` with a
+runtime field is faster than retrieving values from `_source` because of how
+data is loaded from Lucene.
+
+However, there are cases where retrieving fields from `_source` is necessary.
+For example, `text` fields do not have `doc_values` available by default, so you
+have to retrieve values from `_source`. In other instances, you might choose to
+not index specific fields or choose to disable `doc_values` on an index.
+
+With `doc_values` disabled, you could have a field in your index mapping where
+`index` is `true` and `doc_values` is `false`. This combination can be
+confusing because although the field isn't indexed, it is _searchable_. In this
+case, define a runtime field without a script to retrieve values directly
+from `_source`.
```

Comment:
I find that this sentence, while it's trying to clarify things, adds even more confusion :) I would remove it.